RS_max_iter:            250
RS_min_inliers:         3
RS_dist_thresh:         0.02
RANSAC_on_clusters:     True

Voxel_leaf_size:        0.001

# EC_cluster_tolerance:   0.14
EC_cluster_tolerance:   0.05 #tb3
EC_min_size:            8
EC_max_size:            250
